.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 node_js:
-  - "0.10"
   - "4.4.3"
```

Comment:
Does it make sense to test on 4.4.3 if we set engines to 6.9?

Other than that, looks good to me